src/api/mobility/index.ts

```diff
@@ -121,4 +127,53 @@ export default (server: Hapi.Server) => (service: IMobilityService) => {
       return (await service.getBikeStation(payload, h.request)).unwrap();
     },
   });
+
+  /**
+   * Parking violations reporting
+   */
+  server.route({
+    method: 'GET',
+    path: '/bff/v2/mobility/violations-reporting/init',
+    options: {
+      tags: ['api', ' mobility', 'violations'],
```

Comment:
Noticed you have a stray whitespace here, and I'm not sure how well swagger will handle that. Same with the other endpoints.